server/config.go

```diff
@@ -17,6 +17,12 @@ import (
 	"github.com/miekg/dns"
 )
 
+const (
+	SCacheCapacity = 10000
```

Comment:
Do these still need to be exported? I didn't care before, because everything was in the main namespace